resources/wiki-overrides.properties

```diff
@@ -17,19 +14,16 @@ chef-tracking=https://wiki.jenkins-ci.org/display/JENKINS/Chef+Tracking+Plugin
 chosen-views-tabbar=https://wiki.jenkins-ci.org/display/JENKINS/Chosen+Views+Tab+Bar
 ## Source seems to *not* be under jenkinsci org: https://github.com/ThoughtsOnMobile/cocoapods-jenkins-integration
 cocoapods-integration=https://wiki.jenkins-ci.org/display/JENKINS/CocoaPods+Plugin
-cucumber-reports=https://wiki.jenkins-ci.org/display/JENKINS/Cucumber+Reports+Plugin
```

Comment:
https://github.com/jenkinsci/cucumber-reports-plugin/commit/50cb26da9e42955b98ee19f96896175e5b5ac05b#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R16 references the repo now, but the wiki page was just a placeholder, so this is a good improvement.